rust/rope/benches/edit.rs

```diff
@@ -72,7 +72,6 @@ fn benchmark_char_insertion_one_line_edit(b: &mut Bencher) {
     let mut offset = 100;
     b.iter(|| {
         text.edit(offset..=offset, "a");
-        offset += 1;
```

Comment:
maybe it still makes sense to do some loop inside the `iter`? we can just make it some fixed length.